### ISSUE TITLE
Plugin DSP and UI binaries do not need to be split

### ISSUE THREAD

**Issue by falkTX** [sjaehn/lv2tutorial#2]
Another correction.

Having the plugin dsp stuff in a binary and the UI in another binary is not a requirement.
Developers are free to place everything inside a single binary, even multiple plugins and multiple UIs.

You can also split the binaries inside an LV2 bundle so each plugin in the bundle gets a different one.

In resume, there is no restrictions for this whatsoever, and implying there is, is misleading.

**Comment by sjaehn** [comment]
You are right, of course. There is no need to have two ore even more binaries. And I know that you don't do splitting ;-). But you CAN do it. I used LittleFly by brummer10 as an example it the video 01. And he's one of the programmers who prefer to split DSP and GUI code into two binaries.

**Comment by falkTX** [comment]
The splitting makes sense typically, but not always.
I most definitely do it as much as possible. In some stuff like JUCE it is not possible by design, sadly.
